### DISCUSSION TITLE
Disable cursor input outside of playfield on mobile

### DISCUSSION THREAD

**Discussion by hrfarmer** [ppy/osu#26686] [Ideas] [answered]
There's an issue I have whenever I try to play on my iPad, which is that the finger(s) that I use to tap (outside of the playfield) will keep moving the cursor off of my apple pencil, sometimes making me miss, and also being extremely distracting.

https://github.com/ppy/osu/assets/58487401/7b3d982a-f678-4a1e-96d1-e1eb9941efad

A solution I thought of would be that cursor input outside of the playfield could be disabled during gameplay, while still allowing for key inputs to hit streams. If not, I'm just wondering if anything else can be done about this.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22556
